install/ontology/general.rdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
```

Comment:
Such metadata ontologies will require search capabilities. In such a context, I would also include the triples indicating the system how these properties behave in terms of SOLAR search.